base/array.jl

```diff
@@ -1781,8 +1783,12 @@ CartesianIndex(2, 1)
 """
 function findprev(A, start)
     i = start
-    while i >= first(keys(A))
+    f = first(keys(A))
+    i < f && return nothing
+    while true
         A[i] && return i
+        i == f && break
+        # prevind(A, l) can throw/underflow
```

Comment:
Simple copy-paste comment here and below.  Not sure these comments are needed in the first place.

```suggestion
        # prevind(A, f) can throw/underflow
```